### PR TITLE
Synced EN types_strings and search categories

### PR DESCRIPTION
**Pull request by map-per**
Synced EN types_strings and search categories to solve #8290 for the English language.

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -5723,7 +5723,7 @@ zh-Hans:听力学家
 zh-Hant:聽力學|聽力學家
 
 healthcare-blood_donation
-en:Blood donation
+en:Blood donation|Blood Donation Center|Blood Donation Centre
```
Comment (vng) [inline data/categories.txt:5726]: I'd prefer not to add "Center/Centre" here.

File: data/categories.txt [hunk 2]
```diff
@@ -5880,7 +5880,7 @@ zh-Hans:心理治疗师
 zh-Hant:心理治療|心理治療師
 
 healthcare-sample_collection
-en:Sample collection
+en:Sample Collection Centre|Sample Collection
```
Comment (vng) [inline data/categories.txt:5883]: I'd prefer not to add "Center/Centre" here.

File: data/categories.txt [hunk 3]
```diff
@@ -13901,7 +13900,7 @@ mr:पवनचक्की
 # mr:बंकर
 
 natural-cave_entrance
-en:Cave
+en:Cave Entrance
```
Comment (vng) [inline data/categories.txt:13903]: I'd prefer not to add "Entrance" here.

File: data/categories.txt [hunk 4]
```diff
@@ -13975,8 +13974,7 @@ fa:اتشفشان
 mr:ज्वालामुखी
 
 office-estate_agent
-en:Realtor|4Real Estate
-en-GB:4Estate Agent
+en:4Estate Agent|Realtor|Real Estate
```
Comment (vng) [inline data/categories.txt:13977]: Keep 4Real

File: data/categories.txt [hunk 5]
```diff
@@ -15451,7 +15449,7 @@ zh-Hant:山屋
 fa:کلبه‌ی صحرایی
 
 tourism-gallery|@category_tourism
-en:3Gallery|museum
+en:3Art Gallery|Gallery|museum
```
Comment (vng) [inline data/categories.txt:15452]: Keep 3Gallery

**Review by vng — COMMENTED** [review]
I already wrote in some PRs before about nuances on how search works now.
Short story long - avoid common tokes (like Center, etc) if possible.

**Comment by map-per** [comment]
Thanks for the review, I updated the PR to include all suggestions.

**Review by vng — APPROVED** [review]

**Pull request merged**